Presentation/Renderer/BookFormatters/DetailedBookFormatter.cs

```diff
@@ -0,0 +1,15 @@
+using Opcion1LosBorbotones.Domain;
+
+namespace Opcion1LosBorbotones.Presentation.Renderer.BookFormatter;
+
+public class DetailedBookFormatter : IEntityFormatter<Book>
+{
+    public DetailedBookFormatter(Book entity) : base(entity)
+    {
+    }
+
+    public override string ToString()
+    {
+        return $"Title: {_entity.Title}, Author: {_entity.Author}, ISBN: {_entity.Isbn}, Genre: {_entity.Genre}, Publication Year: {_entity.PublicationYear:yyyy-MM-dd}";
```

Comment:
Since it is the presentation layer, can you format it better using Spectre Console so that it is displayed in the console?

Reply:
Done